simbiota/src/detection_system.rs

```diff
@@ -343,49 +334,44 @@ impl DetectionSystem {
         let filename = maybe_filename.unwrap_or_else(|| "<n/a>".to_string());
         let orig_fname = filename.clone();
 
+        let logname = filename.clone();
```

Comment:
IMO this is unnecessary as the log macros try to take values by ref

Reply:
done